packages/php-wasm/compile/php/proc_open7.4.c

```diff
@@ -355,6 +355,7 @@ PHP_FUNCTION(proc_open)
 	struct php_proc_open_descriptor_item *descriptors = NULL;
 	int ndescriptors_array;
 	char **argv = NULL;
+    int nargv_array = 0;
```

Comment:
This should be a bit easier to understand:

```suggestion
    int num_argv = 0;
```

Reply:
Ok! I tried to follow the same logic as in the php legacy code for the descriptors. They used `ndescriptors_aray` for descriptors. But I will change this!